Doc/library/functions.rst

```diff
@@ -1262,7 +1262,8 @@ are always available.  They are listed here in alphabetical order.
    closest multiple of 10 to the power minus *ndigits*; if two multiples are
```

Comment:
I've made a few changes trying to clarify the wording for `round`.  I hope I didn't overstep any bounds.  I researched other, similar functions and looked at the help().  In the first line I wanted to say 'Return the real number *x*', but float() didn't specify real number, even though they both don't work on Complex.

Anyway, this is my first pull, so I hope I did OK.  :-)